### ISSUE TITLE
Fix violations flagged by a11y add on

### ISSUE THREAD

**Issue by el7h1rw2** [Royal-Navy/design-system#953]
Browse to https://storybook.royalnavy.io/ and there will be violations in the `Accessibility` tab. These need fixing across all components.

Update: Closing in favour of individual tickets.

**Comment by hxltrhuxze** [comment]
Closing in favour of individual tickets.
